packages/core/core/src/services/entity-validator/index.ts

```diff
@@ -315,7 +316,11 @@ const createValidateEntity = (createOrUpdate: CreateOrUpdate) => {
 
     const validator = createModelValidator(createOrUpdate)(
       { model, data, entity },
-      { isDraft: options?.isDraft ?? false }
+      {
+        isDraft: options?.isDraft ?? false,
+        // TODO should we pass the default locale here?
+        locale: options?.locale ?? 'en',
```

Comment:
locale should be null by default if not provided (locale is null for content types with disabled i18n). Whoever is using this the entity validator should provide the appropiate locale, and if not I would say it should be empty by default.